habitat/plan.sh

```diff
@@ -125,10 +143,43 @@ do_install() {
 
     build_line "** fixing binstub shebangs"
     fix_interpreter "${pkg_prefix}/vendor/bin/*" "$_chef_client_ruby" bin/ruby
+
     for gem in chef-bin chef inspec-core-bin ohai; do
       build_line "** generating binstubs for $gem with precise version pins"
       "${pkg_prefix}/vendor/bin/appbundler" $CACHE_PATH $pkg_prefix/bin $gem
     done
+
+    build_line "** copying monkey_patch_ignored.rb into package"
+    cp "${CACHE_PATH}/monkey_patch_ignored.rb" "$pkg_prefix/vendor/"
+
+    build_line "** configuring bundle to load monkey_patch"
+    mkdir -p "$pkg_prefix/vendor/bundler/setup"
+    cat <<'EOF' > "$pkg_prefix/vendor/bundler/setup.rb"
+# preload monkeypatch and continue loading bundler
+require_relative '../monkey_patch_ignored'
+require 'bundler/setup'
+EOF
+
+    cat <<EOF > "$pkg_prefix/bin/bundler"
+#!/bin/sh
+RUBYLIB="$pkg_prefix/vendor" exec ruby -r"bundler/setup" "$(hab pkg path core/bundler)/bin/bundler" "\$@"
+EOF
```

Comment:
want to make sure `core/bundler` is included in build deps or else `hab pkg path core/bundler` will not resolve.